query-grammar/src/query_grammar.rs

```diff
@@ -62,6 +62,20 @@ fn word<'a>() -> impl Parser<&'a str, Output = String> {
         })
 }
 
+// word variant that allows more characters, e.g. for range queries that don't allow field
+// specifier
+fn relaxed_word<'a>() -> impl Parser<&'a str, Output = String> {
+    (
+        satisfy(|c: char| {
+            !c.is_whitespace() && !['`', '{', '}', '"', '[', ']', '(', ')'].contains(&c)
```

Comment:
what is special about the backquote?

Reply:
I wasn't sure, I think we can remove it?

Reply:
I copied it from `word`, and kept it since I don't know why it's there. I would keep it for now, we can always remove it later